scripts/get-user.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+"""Outputs details of a random active unlocked user with a given role.
+
+For supplier users, providing additional <framework> and <lot> arguments allows
+filtering by suppliers who have published services on the given framewok and lot.
```

Comment:
Good shout, made them optional so now you can ask for a completely random supplier user as well.